xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -2670,6 +2674,43 @@ void CActiveAE::LoadSettings()
   m_settings.silenceTimeoutMinutes = settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_STREAMSILENCE);
 }
 
+void CActiveAE::ValidateOutputDevices(bool saveChanges)
+{
+  bool changed{false};
+
+  std::string device = m_sink.ValidateOuputDevice(m_settings.device);
```

Comment:
Probably also needs a passthrough true/false parameter, see in CActiveAESink::EnumerateOutputDevices() how all device PCM and passthrough capabilities are tested.
We don't want to fallback passthrough to a non-passthrough capable device.

Reply:
Good point, really is need with the latest changes